src/renderer/components/layout/sidebar.tsx

```diff
@@ -191,17 +212,25 @@ export class Sidebar extends React.Component<Props> {
             >
               {this.renderCustomResources()}
```

Comment:
I just noticed that there is no place here that renders sub menus registered to Lens native menus. Is that still on the table for this PR?

Reply:
Out-of-scope, it needs more refactoring of Lens internals.

Reply:
Makes sense